buildDockerImages.pl

```diff
@@ -29,8 +29,11 @@ sub usage {
     print "                      This must be provided if --private is used.\n";
     print "     --port :         This is the port number for the private registry.\n";
     print "                      This is only used with --private.\n";
-    print "     --proxy :        This is the url of the http(s) proxy to use when accessing the internet.\n";
-    print "                      The proxy is currently only used for images that use curl in their Dockerfiles.\n";
+		print "     --https_proxy :  This is the url of the https proxy to use when accessing the internet.\n";
+		print "                      The proxy is currently only used for images that use curl in their Dockerfiles.\n";
+    print "                      If required by your proxy, the url should include the port, username, and password.\n";
+		print "     --http_proxy :   This is the url of the http proxy to use when accessing the internet.\n";
+		print "                      The proxy is currently only used for images that use curl in their Dockerfiles.\n";
```

Comment:
minor point, but the change from spaces to tabs makes this block less clean depending on editor.    Can it be all spaces or all tabs at the start of these lines?